lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = translate('errors.session.title');
+    content.message = translate('errors.session.message');
+  }
+
+  if (err.code === 'NO_COOKIES') {
+    err.status = 403;
+    content.title = translate('errors.cookies-required.title');
+    content.message = translate('errors.cookies-required.message');
+  }
+
+  content.title = content.title || translate('errors.default.title');
+  content.message = content.message || translate('errors.default.message');
+
+  if (debug === true) {
+    content.message = err.message || err || content.message;
+  }
+
+  return content;
+};
+
+module.exports = (translate, options) => {
+
+  const logger = (options || {}).logger;
```

Comment:
It would be possible to make the translate function optional too, only invoke it if defined.
```
content.title = content.title || (translate && translate('errors.default.title')) || '';
```